src/thunderbots/software/ai/passing/evaluation.h

```diff
@@ -0,0 +1,18 @@
+// TODO: start of file comment
+
+// TODO: if this file (or the corresponding `.cpp`) gets too big, they may need to be broken up
+#pragma once
+
+#include "ai/world/field.h"
+#include "geom/point.h"
+
+/**
+ * Calculates the static position quality for a given position on a given field
```

Comment:
We should provide a better explanation of what "static position quality" is here.

Eg. "Static position quality prefers good passing points on the field from the perspective of a "real soccer player". For example, passing in front of or towards your own net is less desirable than passing near the enemy's net"

Reply:
added that basically verbatiim